@storybook/react types dependency @types/webpack-env conflicting with @types/node

**Describe the bug**
`@storybook/react` types dependency `@types/webpack-env` conflicting with `@types/node`

```bash
node_modules/@types/webpack-env/index.d.ts:282:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'require' must be of type 'Require', but here has type 'NodeRequire'.
```
**To Reproduce**
Steps to reproduce the behavior:
1. Setup a TS env
2. Install @storybook/react
3. Install @types/node
4. Compile project

**Screenshots**
<img width="1541" alt="Screenshot 2019-12-24 at 4 27 34 PM" src="https://user-images.githubusercontent.com/7279475/71410213-63bde680-266a-11ea-83f5-ee3b6825176f.png">

**System:**
```
System:
    OS: macOS 10.15.1
    CPU: (8) x64 Intel(R) Core(TM) i7-3635QM CPU @ 2.40GHz
  Binaries:
    Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v10.16.0/bin/npm
  Browsers:
    Chrome: 79.0.3945.88
    Safari: 13.0.3
```

**Additional context**
Manually editing @types/webpack-env solves this problem.
I tried adding "types" to tsconfig.json to limit type definition scope.
I am using storybook in monorepo setup using yarn.

--- a/app/react/package.json
+++ b/app/react/package.json
@@ -40,7 +40,7 @@
     "@storybook/core": "5.3.0-rc.12",
     "@storybook/node-logger": "5.3.0-rc.12",
     "@svgr/webpack": "^4.0.3",
-    "@types/webpack-env": "^1.13.7",
+    "@types/webpack-env": "^1.15.0",
     "babel-plugin-add-react-displayname": "^0.0.5",
     "babel-plugin-named-asset-import": "^0.3.1",
     "babel-plugin-react-docgen": "^4.0.0",
@@ -57,7 +57,7 @@
   },
   "devDependencies": {
     "@types/mini-css-extract-plugin": "^0.8.0",
-    "@types/node": "^12.12.11",
+    "@types/node": "^13.1.5",
     "@types/webpack": "^4.41.0"
   },
   "peerDependencies": {

